Crashlytics/Crashlytics/Controllers/FIRCLSExistingReportManager.m

```diff
@@ -52,15 +58,15 @@ - (instancetype)initWithManagerData:(FIRCLSManagerData *)managerData
   return self;
 }
 
-NSInteger compareOlder(FIRCLSInternalReport *reportA,
+NSInteger compareNewer(FIRCLSInternalReport *reportA,
```

Comment:
Should there be a comment describing which report is newer, and is the type NSComparisonResult more descriptive?